packages/modules/region/src/services/region-module.ts

```diff
@@ -129,6 +130,25 @@ export default class RegionModuleService
     return result
   }
 
+  @InjectManager("baseRepository_")
+  // @ts-ignore
+  async softDeleteRegions(
```

Comment:
Let's just add a comment that it is not an idempotent operation, so it cannot be reverted simply by restoring the region